antarest/study/storage/variantstudy/variant_study_service.py

```diff
@@ -118,8 +130,10 @@ def get_command(self, study_id: str, command_id: str, params: RequestParameters)
         study = self._get_variant_study(study_id, params)
 
         try:
-            index = [command.id for command in study.commands].index(command_id)  # Maybe add Try catch for this
-            return t.cast(CommandDTOAPI, study.commands[index].to_dto().to_api())
+            index = [command.id for command in study.commands].index(command_id)
+            command = study.commands[index]
+            user_name = self._get_user_name_from_id(command.user_id) if command.user_id else None
+            return t.cast(CommandDTOAPI, command.to_dto().to_api(user_name))
```

Comment:
Instead of casting, you can type: `command: CommandBlock = study.commands[index]` and mypy will understand the code so you won't have to type